manifests/x/Xcas/Xcas/1.6.0/Xcas.Xcas.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Xcas.Xcas
+PackageVersion: 1.6.0
+PackageLocale: en-US
+Publisher: Xcas
+PackageName: Giac/Xcas

```

```suggestion
PackageName: Xcas for Windows
```
This must match Control Panel.